recipes/nest/activate.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+NEST_INSTALL_DIR=$PREFIX
+NEST_DATA_DIR="$NEST_INSTALL_DIR/share/nest"
+NEST_DOC_DIR="$NEST_INSTALL_DIR/share/doc/nest"
+NEST_MODULE_PATH="$NEST_INSTALL_DIR/lib/nest"
+NEST_PYTHON_PREFIX="$NEST_INSTALL_DIR/lib/python3.6/site-packages"
+PYTHONPATH="$NEST_PYTHON_PREFIX:$PYTHONPATH"
+PATH="$NEST_INSTALL_DIR/bin:$PATH"
```

Comment:
There shouldn't be any need to modify the `PATH` (imagine if every package did this).
```suggestion
```

Reply:
Sorry, but it is needed.

Reply:
Again why? `conda activate` prefixes `PATH` with `${PREFIX}/bin` which is exactly what this is doing.

Reply:
You're absolutely right.